scenarios/kubernetes_build.py

```diff
@@ -85,7 +85,7 @@ def main(args):
 
     This is a python port of the kubernetes/hack/jenkins/build.sh script.
     """
-    if os.path.split(os.getcwd())[-1] != 'kubernetes' and \
+    if os.path.split(os.getcwd())[-1] != 'kubernetes' or \
```

Comment:
This change doesn't make sense. This will always error since the path can't be both..?